internal/goobj/read.go

```diff
@@ -679,8 +679,6 @@ func (r *objReader) parseObject(prefix []byte) error {
 			f.FuncData = make([]FuncData, r.readInt())
 			for i := range f.FuncData {
 				f.FuncData[i].Sym = r.readSymID()
-			}
-			for i := range f.FuncData {
 				f.FuncData[i].Offset = int64(r.readInt()) // TODO
```

Comment:
Please, do not modify the internal packages. They are vendored from the Go project.